modules/react-components/src/components/alert/wizard-alert.tsx

```diff
@@ -31,6 +31,10 @@ interface WizardAlert {
     level: AlertLevels;
 }
 
+interface WizardAlertProps {
+    ["data-componentid"]?: string;
+}
```

Comment:
We already have an interface defined for this in core package, which `IdentifiableComponentInterface`. Let's use it here instead of re defining a new interface.

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/6597.